mycroft/messagebus/client/threaded_event_emitter.py

```diff
@@ -0,0 +1,19 @@
+from pyee import EventEmitter
+from multiprocessing.pool import ThreadPool
+
+
+class ThreadedEventEmitter(EventEmitter):
+    """ Event Emitter using the threadpool to run event functions in
+        separate threads using a threadpool.
+    """
+    def __init__(self, threads=None):
+        super().__init__()
+        self.pool = ThreadPool(threads or 10)
+
+    def emit(self, event, *args, **kwargs):
+        """ Override the normal emit launching the function using
+            threadpool.
+        """
+        for f in self._events[event]:
```

Comment:
Will this mess up once events? What if we call `EventEmitter.emit(self, *args, **kwargs)` via the threadpool?

Reply:
I do not think that will get around the issue with the event emitter serializing the actual calls for messages with multiple registered handlers.

This can possibly be done in a simpler and cleaner way by overriding on and once instead using a wrapper to launch the handler in a thread...

Reply:
this messed the once signals, forgot i had merged this to test and it broke get_response for me, any speak message now starts listening

Reply:
I've updated the commit with a wrapper approach instead of overriding emit. This is working well for me so far.